tests/files.rs

```diff
@@ -1,18 +1,30 @@
-use eggcc::util::{Run, TestProgram};
+use std::collections::HashSet;
+
+use eggcc::util::{Run, RunType, TestProgram};
 use insta::assert_snapshot;
 use libtest_mimic::Trial;
 
 fn generate_tests(glob: &str) -> Vec<Trial> {
     let mut trials = vec![];
+
     let mut mk_trial = |run: Run, snapshot: bool| {
+        let snapshot_configurations: HashSet<RunType> = vec![
```

Comment:
Only some of the configurations need snapshots

Reply:
I don't think you need the `vec!`; I think you can just iterate over the array.